plugins/modules/vmware_export_ovf.py

```diff
@@ -70,6 +70,11 @@
     description:
     - Export an ISO image of the media mounted on the CD/DVD Drive within the virtual machine.
     type: bool
+  export_with_extraconfig
+    default: false
+    description:
+    - All extra configuration options are exported for a virtual machine.
+    version_added: '1.18.0'
```

Comment:
```suggestion
    version_added: '2.0.0'
```

As discussed in #1056, I've created a `stable-1.x` branch. New PRs against the `main` branch will make it into [version 2](https://github.com/ansible-collections/community.vmware/milestone/13), not 1.18.